uaclient/cli.py

```diff
@@ -680,8 +711,27 @@ def action_refresh(args, cfg):
     return 0
 
 
-def action_help(_args, _cfg):
-    get_parser().print_help()
+def action_help(args, cfg):
+    service_name = args.service_name
+
+    if not service_name:
+        get_parser().print_help()
+        return 0
+
+    if not cfg:
+        cfg = config.UAConfig()
+
+    help_response = cfg.help(service_name)
+
+    if args.format == "json":
```

Comment:
We need to file a bug to make sure YAML is a supported format option. It's pretty standard in Canonical and we're going to want to focus on that yet still have json as a parsable option.

Reply:
I think I can add this to this PR, I don't think that should be pose any difficulty

Reply:
that's ok, let's do it in follow up because we need to make sure it works and makes sense for all the commands like status/etc.